modules/Layout/Layout.tsx

```diff
@@ -48,6 +49,32 @@ function Layout({ children, description, imageUrl, title, hasError }: PropsWithC
         };
     }, []);
 
+    useEffect(() => {
+        function scrollListener() {
+            if (document.body.scrollTop > 300 || document.documentElement.scrollTop > 300) {
+                setIsScrollToTopVisible(true);
+            } else {
+                setIsScrollToTopVisible(false);
+            }
+        }
+        if (typeof window !== 'undefined') {
+            window.onscroll = scrollListener;
+        }
+
+        return () => {
+            if (window !== null) {
```

Comment:
Why is this check different from the one on line 60? 🤔 
It makes sense to just check for `typeof` in both cases.
Also, the way Next works, `useEffect` should only fire on client, so `window` should always be defined. But good to check nevertheless.